server/spec/conditional_content_spec.rb

```diff
@@ -0,0 +1,258 @@
+require 'spec_helper'
+require 'candlepin_scenarios'
+
+describe 'Conditional Content and Dependent Entitlements' do
+
+  include CandlepinMethods
+
+  before(:each) do
+    @owner = create_owner(random_string('conditional_content_owner'))
+
+    # Three provided products, will be bundled in different ways:
+    @required_product_1 = create_product()
+    @required_product_2 = create_product()
+    @required_product_3 = create_product()
+
+    # This bundled product only contains two of the provided products our conditional content
+    # will require
```

Comment:
Same nit as the other branch "two" -> "three"

Reply:
Fixed.